EnglishFilter/sections/whitelist_stealth.txt

```diff
@@ -29,6 +29,8 @@
 @@||algolia.net/*/indexes/*/queries?x-algolia-agent*vue-instantsearch*algolia-api-key$stealth
 ! https://forum.adguard.com/index.php?threads/34955/ [Stealth Mode - Referrer]
 @@||potokcdn.com^$stealth
+! https://github.com/AdguardTeam/AdguardFilters/issues/41774 [Stealth Mode - Referrer]
+@@||hls.jetcdn.co^$stealth

```

Comment:
Rule is a bit wide. Navigation works fine without the rule for me, just the video not.

@@||hls.jetcdn.co/public/dist/index.html^$stealth

Reply:
It is used to load steam config only